lib/thor/parser/option.rb

```diff
@@ -86,7 +86,7 @@ def usage(padding = 0)
       sample = "[#{sample}]" unless required?
 
       if boolean?
-        sample << ", [#{dasherize("no-" + human_name)}]" unless name == "force"
+        sample << ", [#{dasherize("no-" + human_name)}]" unless name == "force" or name.start_with?("no")
```

Comment:
Good catch. I fixed it and also added a test case for "--nougat" to make sure there will never be a regression :)